packages/types/src/sdk/koa.ts

```diff
@@ -49,3 +60,32 @@ export interface UserCtx<RequestBody = any, ResponseBody = any>
 export interface BBContext extends Ctx {
   user?: ContextUser
 }
+
+export interface ContextEmitter {
```

Comment:
This is really nice, I was actually confused about this the other day when trying to track down some emitter issues - the fact we had the AutomationEmitter and the standard emitter etc.  Really nice stuff!